cli/medperf/commands/dataset/create.py

```diff
@@ -150,22 +151,33 @@ def run_cube_tasks(self):
 
         # Run the tasks
         self.ui.text = "Running preparation step..."
-        self.cube.run(
-            self.ui, task="prepare", timeout=prepare_timeout, **prepare_params
-        )
-        self.ui.print("> Cube execution complete")
+        try:
+            self.cube.run(
+                self.ui, task="prepare", timeout=prepare_timeout, **prepare_params
+            )
+            self.ui.print("> Cube execution complete")
 
-        self.ui.text = "Running sanity check..."
-        self.cube.run(
-            self.ui, task="sanity_check", timeout=sanity_check_timeout, **sanity_params
-        )
-        self.ui.print("> Sanity checks complete")
+            self.ui.text = "Running sanity check..."
+            self.cube.run(
+                self.ui,
+                task="sanity_check",
+                timeout=sanity_check_timeout,
+                **sanity_params,
+            )
+            self.ui.print("> Sanity checks complete")
 
-        self.ui.text = "Generating statistics..."
-        self.cube.run(
-            self.ui, task="statistics", timeout=statistics_timeout, **statistics_params
-        )
-        self.ui.print("> Statistics complete")
+            self.ui.text = "Generating statistics..."
+            self.cube.run(
+                self.ui,
+                task="statistics",
+                timeout=statistics_timeout,
+                **statistics_params,
+            )
+            self.ui.print("> Statistics complete")
+        except RuntimeError as e:
+            cleanup_paths = [out_datapath, out_labelspath]
+            cleanup(cleanup_paths)
+            raise e
```

Comment:
Why do we raise the error here, instead of using `pretty_error`?

in `result.create`:

```
except RuntimeError as e:
    logging.error(f"MLCube Execution failed: {e}")
    cleanup([preds_path, out_path])
    pretty_error("Benchmark execution failed", self.ui)
```

Reply:
Ups just noticed the inconsistency. I agree, let's use `pretty_error` in all scenarios